### DISCUSSION TITLE
Snooze reminders

### DISCUSSION THREAD

**Discussion by bsaunder2002** [iSoron/uhabits#768] [Feature Requests]
All of my todo's are in Habit.  Brush teeth, eat lunch, check email, exercise, meditate, ...

I'd like the ability to snooze an alert/reminder.  Ie if my reminder is set for 12pm, I'd like the popup notification to allow snoozing for a certain amount of time.  Hit the 10m snooze and another reminder will pop up in 10 minutes.  Snooze can be repeated.

**Comment by iSoron** [comment]
Thank you for the feature request, @bsaunder2002, but Loop already has this feature. When a reminder appears, you should be able to see "Yes", "No" and "Later". Clicking later allows you to pick a snooze delay. See the screenshots below.

![Screenshot_1612842533](https://user-images.githubusercontent.com/4699657/107313223-b79ab300-6a57-11eb-9fc8-cfafd806f46c.png)
![Screenshot_1612842614](https://user-images.githubusercontent.com/4699657/107313231-bc5f6700-6a57-11eb-8793-b9ccdab255cd.png)
